docs/content/guides/dagster-pipes/kubernetes.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: "Integrating Kubernetes with Dagster Pipes | Dagster Docs"
+description: "Learn to integrate Dagster Pipes with Kubernetes to launch external code from Dagster assets."
+---
+
+# Integrating Kubernetes with Dagster Pipes
+
+<Note>
+  <strong>Heads up!</strong> This guide focuses on using an out-of-the-box
+  Kubernetes resource. For further customization, use the{" "}
+  <a href="/guides/dagster-pipes/customizing-dagster-pipes-protocols">
+    <code>open_pipes_session</code> approach
+  </a>{" "}
+  instead.
+</Note>
+
+In this guide, we’ll show you how to use [Dagster Pipes](/guides/dagster-pipes) with Dagster’s Kubernetes integration to launch Kubernetes pods and execute external code.
+
+Pipes allows your code to interact with Dagster outside of a full Dagster environment. Instead, the environment only needs to contain `dagster-pipes`, a single-file Python package with no dependencies that can be installed from PyPI or easily vendored. `dagster-pipes` handles streaming `stdout`/`stderr` and Dagster events back to the orchestration process.
+
+**Note**: Dagster Pipes is currently **experimental**.
+
+---
+
+## Prerequisites
+
+To use Dagster Pipes with Kubernetes, you’ll need:
+
+- **In the orchestration environment**, you'll need to install the following packages:
+
+  ```shell
+  pip install dagster dagster-webserver dagster-k8s
+  ```
+
+  Refer to the [Dagster installation guide](/getting-started/install) for more info.
+
+- **A Kubernetes cluster**. This can be an existing cluster. Or, if working locally, you can use [kind](https://kind.sigs.k8s.io/) or [Docker Desktop](https://docs.docker.com/desktop/kubernetes/).
+
+---
+
+## Step 1: Define the external Kubernetes code container
+
+In this step, you’ll create a Kubernetes container that runs some code that uses `dagster-pipes`.
```

Comment:
"Kubernetes container" isn't quite right
Kubernetes is a thing that orchestrates / runs containers but isn't a type of container
"create a container image" is probably the right phrasing